Configurations/unix-Makefile.tmpl

```diff
@@ -34,65 +63,83 @@ MINOR={- $config{minor} -}
 SHLIB_VERSION_NUMBER={- $config{shlib_version} -}
 SHLIB_TARGET={- $target{shared_target} -}
 
-LIBS={- join(" ", map { platform->staticlib($_) // () } @{$unified_info{libraries}}) -}
-SHLIBS={- join(" ", map { platform->sharedlib($_) // () } @{$unified_info{libraries}}) -}
-SHLIB_INFO={- join(" ", map { my $x = platform->sharedlib($_);
-                              my $y = platform->sharedlib_simple($_);
-                              $x ? "\"$x;$y\"" : () }
-                        @{$unified_info{libraries}}) -}
-MODULES={- join(" ", map { platform->dso($_) } @{$unified_info{modules}}) -}
-PROGRAMS={- join(" ", map { platform->bin($_) } @{$unified_info{programs}}) -}
-SCRIPTS={- join(" ", @{$unified_info{scripts}}) -}
+LIBS={- join_filled(" ", "\\", 5, 78,
```

Comment:
maybe add a global $MARGIN or something instead of 78 throughout?

Reply:
The well known environment variable is `$COLUMNS`, and yeah why not

Reply:
Yes, please, COLUMNS is the right name, good.